kisanola/tech-community-frontend
# (feat) add more categories for post

#### What does this PR do?
Add 2 more categories on a post creation

#### Description of Task to be completed?

- Refractor the code of adding categories, create 2 categories

#### How should this be manually tested?

Open the browser and confirm that you can see  2 categories which are mentorship requests or collaborations.

Try to create a post and verify if it will work with the new categories created.

#### Screenshots (if appropriate)

<img width="379" alt="Screen Shot 2020-05-10 at 20 19 52" src="https://user-images.githubusercontent.com/18573157/81508694-dff95380-9305-11ea-8961-f6177d2c1eba.png">

#### Questions:
- what do you guys think about this?

Fix #45
